src/timepicker-select/timepicker-select.component.ts

```diff
@@ -10,7 +10,7 @@ import { Select } from "../select/select.component";
 @Component({
 	selector: "ibm-timepicker-select",
 	template: `
-			<label *ngIf="!skeleton" [attr.for]="id" class="bx--label bx--visually-hidden">{{label}}</label>
+			<label *ngIf="!skeleton" [attr.for]="id" class="bx--label bx--visually-hidden" [innerHtml]="label"></label>
```

Comment:
Same issue as the other PR, if we want to allow arbitrary template content, we should take a template as an input - not raw HTML